Homework/Marianna/MethodsExercises/DigitReversedOrder/Program.cs

```diff
@@ -0,0 +1,54 @@
+﻿ using System;
+
+namespace DigitReversedOrder
+{
+    class Program
+    { /*
+        static void Main()
+        {
+            Console.Write("Enter the digit: ");
+            if (!decimal.TryParse(Console.ReadLine(), out decimal n))
+            {
+                Console.WriteLine("The wrong input! ");
+                return;
+            }
+            Console.WriteLine("Reverse order: "+ ReverseNumber(n));
+        }
+        static decimal ReverseNumber(decimal n)
+        {
+            string temp = n.ToString();
+            string reverse = "";
+            for (int i = temp.Length - 1; i >= 0; i--)
+                reverse += temp[i];
+            return decimal.Parse(reverse);
+        }
+    }
+}
+        */
+        public static void ReversedOrder(decimal n)
+        {
+            decimal reverse = 0;
+            do
+            {
+                n *= 10;
+                reverse = reverse * 10 + n % 10;
+                n /= 10;
+            }
+            while (n !=0);
```

Comment:
usually `while` in `do {...} while` loops is on the same line as the closing bracket